mmcv/ops/csrc/pytorch/npu/fused_bias_leakyrelu_npu.cpp

```diff
@@ -0,0 +1,55 @@
+#include "pytorch_npu_helper.hpp"
+
+using namespace NPU_NAME_SPACE;
+using namespace std;
+
+Tensor fused_bias_leakyrelu_op_impl(Tensor& input, Tensor& bias,
+                                    Tensor& refer, int act,
+                                    int grad, float alpha, float scale);
+
+Tensor fused_bias_leakyrelu_npu(Tensor& input, Tensor& bias,
+                                Tensor& refer, int act,
+                                int grad, float alpha, float scale){
+  at::Tensor py = at::empty_like(input);
+  //forward
+  if (grad == 1){
```

Comment:
According to
https://github.com/open-mmlab/mmcv/blob/c69d5c4ce19efd5d597eeb6f5654dc170a74943e/mmcv/ops/fused_bias_leakyrelu.py#L174

```python
// forward
grad == 0

// backward
grad == 1
```

Reply:
And `act` is not used here.

Reply:
> And `act` is not used here.
According to [mmcv/mmcv/ops/fused_bias_leakyrelu.py], act is always 3.